### ISSUE TITLE
"No known servers" error with invite links

### ISSUE THREAD

**Issue by ethindp** [spantaleev/matrix-docker-ansible-deploy#1201]
I'm trying to invite a friend into a room on my Matix server and he's getting the error "No known servers". I'm getting around this with matrix-registration but I'd prefer to fix this. I have port 8448 open on my server. What should I investigate to fix this problem? My research isn't yielding any actual solution.

**Comment by aaronraimist** [comment]
Can you show the link you used?

**Comment by ethindp** [comment]
Yeah. Its this one: https://matrix.to/#/!QFLXCqIGGAVsafilFe:the-gdn.net?via=the-gdn.net

**Comment by aaronraimist** [comment]
Are you sure you included the `?via=the-gdn.net` bit on the end (and that they didn't remove it)?

That's an important part of the URL which will cause the No known servers error if it is missing.

**Comment by ethindp** [comment]
I'm pretty sure they didn't remove it. I've asked him though and if he did I'll update this issue, but for now assume that he didn't.
